README.md

```diff
@@ -1,11 +1,283 @@
-# OneMobileSDK Examples & Playgrounds
+# O2 Mobile SDK Tutorial – for iOS and tvOS <a name="head"></a>
 
-This repository contains a lot of examples and samples of OneMobileSDK usage.
+Welcome to the O2 Mobile SDK (OMSDK or SDK). The purpose of this document is to provide you with everything you need to know to get started with and use the O2 Mobile SDK. The audience for this document is an experienced iOS Developer with in-depth knowledge of using Xcode, the Swift Programming Language, iOS API programming, and either CocoaPods or Carthage.
```

Comment:
@MarkGerl 
This is not true - about experienced iOS engineers - if we are not including any code snippets - than this is just text.

Reply:
Just need to change ... "...The purpose of this document and the linked source code examples is to provide you with..."